CMakeLists.txt

```diff
@@ -2,7 +2,13 @@ cmake_minimum_required(VERSION 3.7 FATAL_ERROR)
 
 project(cppTango)
 
-set(CXX_STANDARD_REQUIRED 11)
+# C++17 and higher support is currently not possible as omniorb uses
+# throw specifications and these are not supported anymore.
```

Comment:
This comment is actually not true when using omniORB >= 4.2.4

Reply:
@bourtemb I'm fixing that in #725 already.

Reply:
Maybe I'll just remove the comment in this case? Or at least change it to *You need omniORB>=4.2.4 for C++17*. Btw is it 4.2.4 or 4.3?

Reply:
@mliszcz Change it to `You need omniORB>=4.2.4 for C++17.` as both 4.2.2 and 4.3 work with `C++17`.

Reply:
@t-b done! I also linked your conversation with omniORB maintainer in the commit message.

Reply:
@mliszcz Nice!